include/ICursorControl.h

```diff
@@ -160,6 +160,9 @@ namespace gui
 		\param rect: A pointer to an reference rectangle or 0 to disable the reference rectangle.*/
 		virtual void setReferenceRect(core::rect<s32>* rect=0) = 0;
 
+		//! Sets or unsets relative mouse mode

```

Comment:
I'd like a more precise description here.
Looking at the SDL docs the following should be accurate:
`//! Locks mouse in place so that only motion relative to the old position is sent`
right?

Reply:
https://wiki.libsdl.org/SDL_SetRelativeMouseMode
>While the mouse is in relative mode, the cursor is hidden, and the driver will try to report continuous motion in the current window. Only relative motion events will be delivered, the mouse position will not change.

I'm not sure if this description is entirely correct though. I.e. mouse position does change (we depend on that).
One thing that is different is that you can still move the mouse and get relative motion events, even if you are at the boarder of the window.

Reply:
Does the new change of `Internally fixes the mouse position, and reports relative mouse movement compared to the old position` sound good?